recipe/meta.yaml

```diff
@@ -88,7 +88,7 @@ outputs:
         - {{ pin_subpackage("libclang", exact=True) }}
         - {{ pin_subpackage("libclang-cpp", exact=True) }}
         # Use the same requirements as the top-level requirements
-        - libcxx-devel {{ cxx_compiler_version }}   # [osx]
+        - libcxx-devel              # [osx]
```

Comment:
I would have expected the "use same as top-level" to have something to do with having the same hash of the host environment and maybe not busting the cache to rebuild things unnecessarily; however, I'm not sure from https://github.com/conda-forge/clangdev-feedstock/commit/91ad7d6a99bcccf2549c9c5f4297ee067aab6908 resp. https://github.com/conda-forge/clangdev-feedstock/commit/ac155e54149d7c70c74a42c3990dd543186b5c1c if that's actually the case? I've removed it now to try and see.

Reply:
I should probably update the comment `# Use the same requirements as the top-level requirements` for these outputs, but since I don't know/understand the original reason, I can only remove them. If you have input on this, please let me know.

Reply:
Tentatively updating the comment now to be
```
# unpinned because we depend on clangxx here, which pins libcxx-devel
```
If we want to keep everything on the same libcxx version, we could go one step further and pin everything to `{{ version }}`, now that we're building `libcxx` before `clangdev`.